### ISSUE TITLE
Add-on change log is not visible in all the tabs of the store

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#19205]
### Brief summary

When an add-on is updatable the change log is not visible in the updatable and the available tabs of the store.

### Steps to reproduce

1. If you have not installed NVDA Dev & Test Toolbox, install it
2. If you have latest version (8.0), spoof it in order to have a lower version installed.
3. Restart NVDA
4. Open the Add-on Store; you should land in installed add-ons tab
5. Try to locate the What's new item in NVDA Dev & Test Toolbox context menu
6. Go to updatable add-ons tab; NVDA Dev & Test Toolbox should be there
7. Try to locate the What's new item in NVDA Dev & Test Toolbox context menu

### Actual behavior

At steps 5. and 7. the What's new item cannot be found.

### Expected behavior

At steps 5. and 7. you should be able to find the What's new item as in the available add-ons tab

### NVDA type

source copy

### NVDA version

Last master (2274e980574d5746644040bd332fa78c9dfa20eb) or any version after #18816

### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

### Windows version

Windows 11 24H2 (AMD64) build 26100.6899

### Name and version of other software in use when reproducing the issue

N/A

### Other information about your system

N/A

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by CyrilleB79** [comment]
Cc @nvdaes

**Comment by nvdaes** [comment]
@CyrilleB79 , are you sure that the changelog should be visible at this moment, in the updatable and the available tabs? We are still working to integrate it in the store, so that json files are generated with a changelog. See:

https://github.com/nvaccess/addon-datastore-validation/pull/52

If you have tested with a generated json file containing a changelog, please specify it. I tried to test it when working in the store gui, but perhaps I missed something.

**Comment by CyrilleB79** [comment]
Oh sorry, I had not followed this part of the work.

I had just read that add-on change log support had been integrated in NVDA 2026.1's provisional change log and was wondering why it wasn't available in updatable and available add-ons tabs of the store, while it was in the installed add-ons tab.

We may just close this issue or keep it to track here the dependency from the addon-datastore-validation repo.

**Comment by seanbudd** [comment]
I think it's worth keeping the issue open while we work to resolve this

**Comment by CyrilleB79** [comment]
In this case, put it in the 2026.1 milestone while at it.
Because the change log mentions a new feature that is not yet totally effective.

**Comment by nvdaes** [comment]
@CyrilleB79 , nvaccess/addon-datastore-validation#52 has been merged. If you wish and have aadd-on with an changelog not submitted yet to the store, feel free to submit it. Otherwise, when I have ready the translateNvdaaddonsWithCrowdin, I'll submit it. Now the changelog should be shown in the available and updatable tabs, and, when an add-on is presented in the Updatable Add-ons dialog, there the changelog should be also available.

**Comment by seanbudd** [comment]
Closing as fixed by https://github.com/nvaccess/addon-datastore-validation/pull/52
